nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/test/java/org/nuxeo/ecm/automation/core/operations/security/PermissionAutomationTest.java

```diff
@@ -87,14 +101,26 @@ public void initRepo() throws Exception {
 
         when(administratorGroupsProvider.getAdministratorsGroups()).thenReturn(
                 Collections.singletonList("administrators"));
+
+        DocumentModel testUser = userManager.getBareUserModel();
```

Comment:
A faster test with less stuff to deploy + one less maven dependency.

There is no real need to create a user here, you just assume "existingUser" is a real user that exists (+ there are already stuff mocked in the test).